hw/ip/flash_ctrl/data/flash_ctrl.hjson

```diff
@@ -895,7 +895,7 @@
                 name: "ECC_EN",
                 mubi: true,
                 desc: '''
-                  Region is ECC enabled (both integrity and reliability ECC).
+                  Region is ICV and reliability ECC enabled.
```

Comment:
Region is integrity check and reliability ECC are enabled.

Reply:
ah that's better.